core/src/main/java/org/apache/accumulo/core/file/rfile/PrintInfo.java

```diff
@@ -192,23 +268,43 @@ public void execute(final String[] args) throws Exception {
           }
         }
 
+        Formatter formatter = null;
+        if (opts.formatterClazz != null) {
+          final Class<? extends Formatter> formatterClass = getFormater(opts.formatterClazz);
+          FormatterConfig config = new FormatterConfig();
+          config.setPrintTimestamps(true);
+          formatter =
+              FormatterFactory.getFormatter(formatterClass, new FormatterAdapter(dataIter), config);
+        }
```

Comment:
If using BiFunction could possibly do

```suggestion
        BiFunction<Key,Value,String> formatter = null;
        if (opts.formatterClazz != null) {
          final Class<? extends BiFunction<Key,Value,String>> formatterClass = getFormater(opts.formatterClazz);
          formatter = formatterClass.newInstance();
        } else if (opts.fullKeys) {
          formatter = (key, value) - > {key.toStringNoTruncate() + " -> " + value};
        } else if(opts.dump){
           formatter = (key, value) - > {key + " -> " + value};
        }
```